packages/gatsby-tinacms-mdx/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "gatsby-tinacms-mdx",
+  "version": "0.0.1",
+  "main": "index.js",
+  "types": "src/index.d.ts",
+  "license": "Apache-2.0",
+  "files": [
+    "src",
+    "index.js",
+    "gatsby-node.js"
+  ],
+  "keywords": [
+    "tinacms",
+    "cms",
+    "react",
+    "mdx",
+    "gatsby"
+  ],
+  "bugs": {
+    "url": "https://github.com/tinacms/tinacms/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacms/tinacms.git",
+    "directory": "packages/react/gatsby-tinacms-mdx"
+  },
+  "scripts": {
+    "watch": "tinacms-scripts watch",
+    "build": "tinacms-scripts build",
+    "dev": "tinacms-scripts dev",
+    "test": "jest --passWithNoTests"
+  },
+  "devDependencies": {
+    "@tinacms/scripts": "^0.1.11",
+    "@types/jest": "^24.0.15",
+    "@types/js-yaml": "^3.12.1",
+    "jest": "^24.8.0",
+    "tinacms": "^0.12.1-alpha.0",
+    "ts-jest": "^24.0.2"
+  },
+  "dependencies": {
```

I was having trouble getting this branch to build untill I added `"@tinacms/form-builder": "0.2.13",`
```suggestion
  "dependencies": {
    "@tinacms/form-builder": "0.2.13",
```